cli/src/settings.ts

```diff
@@ -2,38 +2,30 @@ const production = process.env.NODE_ENV == 'production'
 const default_api_url = production
   ? 'http://orakl-api.orakl.svc.cluster.local'
   : 'http://localhost:3000/api/v1'
+
 const default_delegator_url = production
   ? 'http://orakl-delegator.orakl.svc.cluster.local'
   : 'http://localhost:3002/api/v1'
 
-const default_fetcher_host = production
-  ? 'http://orakl-fetcher.orakl.svc.cluster.local'
-  : 'http://localhost'
-const default_fetcher_port = production ? '4040' : '3001'
-
 const default_listener_host = production
-  ? 'http://aggregator-listener.orakl.svc.cluster.local'
+  ? 'http://vrf-listener.orakl.svc.cluster.local'
```

Comment:
Seems like we don't have these settings for RR. I'll make an issue for that

Reply:
no you don't need to, it is default host and cli command can take host address as optional parameter

Reply:
okay, just out of curiosity is it mentioned anywhere in the docs or readme?

Reply:
https://docs.orakl.network/node-operators-guide/cli/reporter#list-active-reporters

this is the example cli taking host and port

Reply:
great! thanks for the reference